File: expense-tracker-backend/src/main/java/matthias/expense_tracker/purchase/shops/ShopEntity.java
package matthias.expense_tracker.purchase.shops;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.Entity;
import javax.persistence.Table;

@Getter
@Setter
@Table(name = "purchase_shop")
@Entity
public class ShopEntity extends BaseEntity {

    private String name;
}

File: expense-tracker-backend/src/main/kotlin/matthias/expense_tracker/shop/ShopEntity.kt
package matthias.expense_tracker.shop

import matthias.expense_tracker.common.jpa.BaseEntity
import java.util.*
import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Table

@Entity
@Table(name = "purchase_shop")
class ShopEntity : BaseEntity {

    constructor() : super()
    constructor(id: UUID?) : super(id)

    @Column(columnDefinition = "citext", nullable = false)
    lateinit var name: String
}